exercises/exercise_2.js

```diff
@@ -2,7 +2,7 @@ const characters = require("../data/characters.json");
 
 // TODO: return all characters with known birth year
 const charactersWithKnownBirthDate = () => {
-  // CODE HERE
+  return characters.filter(character => character.birth_year !== "unknown")
 };
-
+console.log(charactersWithKnownBirthDate());
```

Comment:
log big nono

Reply:
need log for evaluation! no say nono to log!